src/controllers/dashboard/dashboard.html

```diff
@@ -1,11 +1,11 @@
 <div id="dashboardPage" data-role="page" class="page type-interior dashboardHomePage fullWidthContent">
     <div class="content-primary">
-        <div class="dashboardSections" style="padding-top:.5em;">
-            <div class="dashboardColumn dashboardColumn-2-60 dashboardColumn-3-46">
```

Comment:
I seem to remember these `dashboardColumn-3-x` classes were unused, but just want to confirm. :smile: